cli/src/main/scala/com/codacy/analysis/cli/Main.scala

```diff
@@ -234,8 +234,9 @@ class MainImpl extends CLIApp {
     }
   }
 
-  private def cleanup(directoryOpt: Option[File]): Unit = {
-    val directory = directoryOpt.getOrElse(File.currentWorkingDirectory) / ".codacy.json"
-    directory.delete(swallowIOExceptions = true)
+  private def cleanup(directory: File): Unit = {
```

Comment:
Instead of cleanup maybe `removeCodacyRuntimeConfigurationFiles` to make it really clear what it does.

Also leave a TODO near the method saying this can be removed when all tools are using the 3.+ seed version.